### PR TITLE
Add clinia as a company that uses Turbo

### PR DESCRIPTION
**Pull request by etiennecl**
Very grateful to be able to have proper tooling and all the work that you guys are putting into Turbo. As we are migrating all of our frontend to Turborepo and all of our upcoming open-source frontend libraries, we would be honored to have our logo on the showcase page.

### CODE REVIEW

**Review by NicholasLYang — APPROVED** [review]

**Comment by mehulkar** [comment]
@etiennecl the prettier check is failing:

```
//:check:prettier: [warn] docs/components/clients/users.ts
```
could you run `prettier -w docs/components/clients/users.ts` and commit / push again?

File: docs/components/clients/users.ts [hunk 1]
```diff
@@ -547,6 +547,12 @@ export const users: Array<TurboUser> = [
     infoLink: "https://hash.dev/",
     pinned: true,
   },
+  {
+    caption: "Clinia",
+    image: "/images/logos/clinia.svg",
+    infoLink: "https://clinia.dev/",
```
Comment (mehulkar) [inline docs/components/clients/users.ts:553]: This URL doesn't resolve for me, is it temporarily down or is there a typo?

**Review by mehulkar — CHANGES_REQUESTED** [review]

**Comment by rakesh-gupta29** [comment]
I think the link should be https://clinia.com/en-ca

**Comment by mehulkar** [comment]
@etiennecl closing this down due to inactivity, but let me know if you want to open it up again!

**Pull request closed without merging**